lib/compat/wordpress-6.1/get-global-styles-and-settings.php

```diff
@@ -5,44 +5,46 @@
  * @package gutenberg
  */
 
-/**
- * Adds global style rules to the inline style for each block.
- */
-function wp_add_global_styles_for_blocks() {
-	$tree        = WP_Theme_JSON_Resolver_Gutenberg::get_merged_data();
-	$block_nodes = $tree->get_styles_block_nodes();
-	foreach ( $block_nodes as $metadata ) {
-		$block_css = $tree->get_styles_for_block( $metadata );
+if ( ! function_exists( 'wp_add_global_styles_for_blocks' ) ) {
+	/**
+	 * Adds global style rules to the inline style for each block.
+	 */
+	function wp_add_global_styles_for_blocks() {
```

Comment:
Instead of checking whether or not the core `wp_add_global_styles_for_blocks` exists, should we instead rename the function to `gutenberg_add_global_styles_for_blocks` so that in the plugin, we're always calling the Gutenberg version? This would make it similar to the function lower down in this file (`gutenberg_get_global_stylesheet`).